.ci/docker/docker-compose.yml

```diff
@@ -101,6 +101,17 @@ services:
       interval: 30s
       timeout: 10s
       retries: 5
+  
+  memcached:
+    image: memcached:alpine
+    user: 11211
```

Comment:
Docker complains about this not being a string. If I wrap it in quotes, it seems to work

Reply:
```suggestion
    user: "11211"
```